### PR TITLE
feat: fe sync packages

### PR DESCRIPTION
**Pull request by adhiraj23zelthy**

### CODE REVIEW

File: frontend/src/mocks/appConfigurationHandlers.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const appConfigurationHandlers = [
 						datetime_format_display: 'August 05 2006, 3:05 PM',
 						logo: '',
 						fav_icon: '',
-						extra_config: null,
+						extra_config: {"git_config":{"branch":{"dev":"dev","prod":"main","staging":"staging"},"repo_url":"https://github.com/Healthlane-Technologies/Zango"},"sync_packages":true},
```
Comment (DevilsAutumn) [inline frontend/src/mocks/appConfigurationHandlers.js:33]: @adhiraj23zelthy are these changes intentional?

Comment (adhiraj23zelthy) [reply frontend/src/mocks/appConfigurationHandlers.js:33]: Yes, it will help us to test sync package functionality in frontend using mock API.

**Review by DevilsAutumn — APPROVED** [review]
